enterprise/api/v1/views.py

```diff
@@ -245,10 +251,6 @@ def enterprise_learners(self, request, pk):
                     else:
                         if already_linked:
                             already_linked_emails.append((email, already_linked.enterprise_customer))
-                        elif email in emails:
```

Comment:
don't see why we would want to add email back to the emails set where it came from so removed

Reply:
unused duplicate_emails

Reply:
@jlajoie want to ensure I am not breaking anything by removing these here ...

Reply:
looks good chatted offline